code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -272,3 +272,41 @@
 	icon_state = "scarecrow_sack"
 	item_state = "scarecrow_sack"
 	flags_inv = HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
+
+/obj/item/clothing/mask/gondola
+	name = "gondola mask"
+	desc = "Genuine gondola fur."
+	icon_state = "gondola"
+	item_state = "gondola"
+	flags_inv = HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/clothing/mask/gondola/speechModification(M)
+	if(copytext(M, 1, 2) != "*")
+		M = " [M]"
+		var/list/spurdo_words = strings("spurdo_replacement.json", "spurdo")
+
+		for(var/key in spurdo_words)
+			var/value = spurdo_words[key]
+			if(islist(value))
+				value = pick(value)
+
+
+			M = replacetextEx(M,regex(uppertext(key),"g"), "[uppertext(value)]")
+			M = replacetextEx(M,regex(capitalize(key),"g"), "[capitalize(value)]")
+			M = replacetextEx(M,regex(key,"g"), "[value]")
+
+		if(prob(30))
+			M += pick(" :"," X")
+			if(prob(20))
+				M += "-"
+			var/smiles = rand(2, 7)
+			while (smiles > 0)
+				M += "D"
+				smiles--

```

Comment:
in all honesty though I'm not sure if smilies would be allowed, since maintainers hate fun things. especially thanks to "XD", and it not being for catgirls.

Reply:
brain dambleg lines have smilies and this is rarer than those so dont worry